src/Symfony/Bundle/FrameworkBundle/Resources/config/messenger.xml

```diff
@@ -103,6 +103,7 @@
             <argument /> <!-- senders service locator -->
             <argument type="service" id="messenger.retry_strategy_locator" />
             <argument type="service" id="logger" on-invalid="ignore" />
+            <argument type="service" id="event_dispatcher" />
```

Comment:
Must be moved to the new PHP config file (the XML file does not exist anymore in current master).